### ISSUE TITLE
VariableOverflow rule does not work for return value of static method

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#946]
Version 2.14.8

I just noticed that the overflow rule does not catch errors with return values of static methods.

```
CLASS Foo:

   METHOD PUBLIC VOID FooMethod(
   ):
      DEFINE VARIABLE lcChar AS CHARACTER NO-UNDO.
      DEFINE VARIABLE liInt AS INTEGER NO-UNDO.

      // Doesnt work
      lcChar = Foo:ReturnLongCharStatic().

      // Works
      lcChar = THIS-OBJECT:ReturnLongChar().

      // Doesnt work
      liInt = Foo:ReturnInt64Static().

      // Works
      liInt = THIS-OBJECT:ReturnInt64().
   END METHOD.

   METHOD PRIVATE STATIC LONGCHAR ReturnLongCharStatic():
      DEFINE VARIABLE llcLongChar AS LONGCHAR NO-UNDO.
      llcLongChar = "Foo":U.
      RETURN llcLongChar.
   END METHOD.

   METHOD PRIVATE LONGCHAR ReturnLongChar():
      DEFINE VARIABLE llcLongChar AS LONGCHAR NO-UNDO.
      llcLongChar = "Foo":U.
      RETURN llcLongChar.
   END METHOD.

   METHOD PRIVATE STATIC INT64 ReturnInt64Static():
      DEFINE VARIABLE llcInt64 AS INT64 NO-UNDO.
      llcInt64 = 1234.
      RETURN llcInt64.
   END METHOD.

   METHOD PRIVATE INT64 ReturnInt64():
      DEFINE VARIABLE llcInt64 AS INT64 NO-UNDO.
      llcInt64 = 1234.
      RETURN llcInt64.
   END METHOD.

END CLASS.
```

**Comment by gquerret** [comment]
Reproduced.

**Comment by gquerret** [comment]
Fixed in develop branch. RCode has to be available to trigger issues.
